test/functional/trustProxy.test.js

```diff
@@ -0,0 +1,87 @@
+'use strict'
+
+const request = require('supertest')
+const {assert} = require('chai')
+const sinon = require('sinon')
+const express = require('express')
+const decache = require('decache')
+
+const userInfo = {
+  emails: [{value: 'test.user@test.com'}],
+  id: '10',
+  userId: '10',
+  _json: {domain: 'test.com'}
+}
+
+describe('Trust Proxy', () => {
+  describe('when trust proxy env is not true', () => {
+    before(() => sinon.stub(express.request, 'isAuthenticated').returns(false))
+    beforeEach(() => decache('../../server/index'))
+    after(() => sinon.restore())
+    // app moved inside tests so that it can be recreated later with different
+    // environment variables setup.
+    var app = require('../../server/index')
```

Comment:
Could we use `const` here instead? Our convention is to avoid the use of `var` and to only use `let` when the reference will be reassigned.

Reply:
Sure.  I just pushed an update that fixes these.